packages/snaps-simulator/package.json

```diff
@@ -11,11 +10,17 @@
     "type": "git",
     "url": "https://github.com/MetaMask/snaps.git"
   },
+  "sideEffects": false,
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
   "scripts": {
-    "build": "webpack --mode production --config-name app",
-    "build:vendor": "webpack --mode production --config-name vendor",
+    "build": "yarn build:tsc && yarn build:webpack",
+    "build:webpack": "yarn webpack --config-name main --config-name test --mode production --progress",
```

Comment:
When/how is the vendor built when using `build:webpack`? 🤔

Reply:
It's not. You have to run `yarn build:vendor` manually at the moment.